src/ActionsImporter/Models/Variable.cs

```diff
@@ -28,5 +29,5 @@ public Variable(string key, Provider provider, string message, string? defaultVa
 
     public string? Placeholder => DefaultValue is not null && IsPassword ? $"({DefaultValue})" : null;
 
-    private Provider Provider { get; }
+    public Provider Provider { get; }
```

Comment:
Yup! I wanted to make it so that the `ProviderNames` could be retrieved programatically. Previously, we were relying on a hardcoded string list.

This public method is used here: https://github.com/github/gh-actions-importer/pull/99/files#diff-d636be41658afaef7332f33a41111411de23eaca955b43de6a8ac7921cc2c26bR30

And `Constants.ProviderNames` is used here https://github.com/github/gh-actions-importer/pull/99/files#diff-51f7db35741780dd90b337c121fb0ebd7242f62adebac623d6d159913f79171dR56 where it used to be a hardcoded list